tests/assert_pass/for_loop_over.yul

```diff
@@ -0,0 +1,14 @@
+{
+	let max := calldataload(0)
+	if gt(max, 10) { revert(0, 0) }
+	let x
+	let i
+	for {} lt(i, max) {
+		x := add(x, 1)
```

Comment:
I mean it doesn't matter, but for the sake of readability, wouldn't you rather swap the post and the body part of this loop?

Reply:
wait, I thought this was the body, is it not?

Reply:
pre condition post body, just like in a c-style for loop

Reply:
yea, I've written a bunch of tests wrong then haha but doesn't really affect the semantics